src/parser_data.h

```diff
@@ -179,7 +179,10 @@ struct ly_in;
 #define LYD_PARSE_JSON_NULL 0x4000000       /**< Allow using JSON empty value 'null' within JSON input, such nodes are
                                                  silently skipped and treated as non-existent. By default, such values
                                                  are invalid. */
-
+#define LYD_PARSE_JSON_STRING_DATATYPES 0x8000000 /*!**< By default, JSON data values are validated to be in the proper format.
```

Comment:
Please use the common `/**<` starting tag. Also, the description could be more exact such as:
```
By default, JSON data values are expected to be in the correct format according to RFC 7951 based on their type. Using this option the validation can be softened to accept boolean and number type values enclosed in quotes.
```
No need to mention *libyang* v1, it is obsolete for a long time now.

Reply:
the starting tag was a typo, fixed